src/App.js

```diff
@@ -1,24 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Form from './components/form/Form';
 
 function App() {
```

Comment:
Change to a class component, please. We should use class components in this homework